tb_plugin/torch_tb_profiler/profiler/node.py

```diff
@@ -121,11 +122,23 @@ def create(cls, event, device_nodes):
 
 class DeviceNode(BaseNode):
     def __init__(self, name, start_time, end_time, type, external_id=None,
-            op_node=None):
+            op_node=None, blocks_per_SM=None, occupancy=None):
         super().__init__(name, start_time, end_time, type, external_id)
         self.op_node = op_node  # The cpu operator that launched it.
+        self.blocks_per_SM = blocks_per_SM
+        self.occupancy = occupancy
 
     @classmethod
     def create(cls, event):
         kwargs = BaseNode.get_node_argument(event)
```

Comment:
merge the two get_node_arguments into DeviceNode.get_node_arguments?

Reply:
Thanks. Done.